Library/Formula/nsq.rb

```diff
@@ -61,16 +61,23 @@ class Nsq < Formula
     url "https://github.com/blang/semver.git",
       :revision => "9bf7bff48b0388cb75991e58c6df7d13e982f1f2"
   end
+  
+  go_resource "github.com/julienschmidt/httprouter" do
+    url "https://github.com/julienschmidt/httprouter.git",
+      :revision => "6aacfd5ab513e34f7e64ea9627ab9670371b34e7"
+  end
 
   def install
     # build a proper GOPATH tree for local dependencies
-    (buildpath + "src/github.com/bitly/nsq").install "internal", "nsqlookupd", "nsqd", "nsqadmin"
+    (buildpath + "src/github.com/nsqio/nsq").install "Makefile", "apps", "internal", "nsqlookupd", "nsqd", "nsqadmin"
 
     ENV["GOPATH"] = buildpath
     Language::Go.stage_deps resources, buildpath/"src"
 
-    system "make"
-    system "make", "DESTDIR=#{prefix}", "PREFIX=", "install"
+    Dir.chdir(buildpath + "src/github.com/nsqio/nsq") do
```

Comment:
Go 1.5 changed the way `internal` packages are evaluated, which meant that more dirs needed to be under `GOPATH`, which meant that we needed to build from within `GOPATH`.

Open to alternatives if this isn't gonna fly...